docs/src/modules/components/AppFrame.js

```diff
@@ -113,40 +109,66 @@ const styles = (theme) => ({
     '@media print': {
       display: 'none',
     },
-  },
-  appBar: {
+  };
+});
+
+const StyledAppBar = styled(AppBar, {
+  shouldForwardProp: (prop) => prop !== 'disablePermanent',
+})(({ disablePermanent, theme }) => {
+  return {
     transition: theme.transitions.create('width'),
-  },
-  language: {
+    ...(!disablePermanent && {
+      boxShadow: 'none',
+    }),
+    ...(!disablePermanent && {
+      [theme.breakpoints.up('lg')]: {
+        width: 'calc(100% - 240px)',
+      },
+    }),
```

Comment:
```suggestion
    ...(!disablePermanent && {
      boxShadow: 'none',
      [theme.breakpoints.up('lg')]: {
        width: 'calc(100% - 240px)',
      },
    }),
```

Reply:
Thanks! Applied manually since the diff looks a bit "busy" and I wasn't sure this actually creates valid JS.